els/categories/api.py

```diff
@@ -0,0 +1,18 @@
+from categories.models import category
+from rest_framework import viewsets, permissions
+from .serializers import categorieserializer
+
+# category Viewset
+
+
+class categoryViewSet(viewsets.ModelViewSet):
+    permission_classes = [
+        permissions.IsAuthenticated,
+    ]
+    serializer_class = categorieserializer
```

Comment:
Change also casing for serializer here. Adjust accordingly
```suggestion
    serializer_class = CategoriesSerializer
```